versions.props

```diff
@@ -25,7 +25,7 @@ io.opentracing:*=0.33.0
 io.prometheus:*=0.2.0
 javax.servlet:javax.servlet-api=3.1.0
 joda-time:joda-time=2.9.9
-junit:junit=4.13.1
+junit:junit=4.13.2
```

Comment:
I'm surprised that we need this here? I'm not against it, just found it odd that our unit version is driven by our Hadoop version.

Reply:
So Hadoop bumped the version from 4.13.1 to 4.13.2 (based on gradle resolution rules) - I just made it explicit here.